docs/api/openapi3.json

```diff
@@ -1153,7 +1169,7 @@
       "url": "https://www.elastic.co/licensing/elastic-license"
     },
     "title": "Infra API",
-    "version": "0.0.0"
+    "version": "99.99.99999"
```

Comment:
Not sure if this version updating is expected

Reply:
It's not, did you generate this file manually, or was it done with the test? The test should fail because it's expecting version 0.

I think we should not tell anyone to use `9.9.9` because that effectively disables versioning for them.

Reply:
I used `make docs`. I'll update that command if we shouldnt be generating it anymore

Reply:
Oh, I did not realize that was still being done.  I guess that's why I had moved it to a test-only location, to not conflict with the file that we generate at runtime for a release.

I think removing it from `make docs` for now makes sense. We'll figure out how to integrate it into the docs later.

Reply:
Fixed in the new commit